.github/workflows/tests-e2e-npm.yml

```diff
@@ -37,16 +41,22 @@ jobs:
         run: |
           echo "Parsing JSON environment variables..."
           echo "$ENV_VARS_JSON" | jq -r 'to_entries|map("\(.key)=\(.value|tostring)")|.[]' >> $GITHUB_ENV
+      - uses: actions/setup-node@v4
+        with:
+          node-version: ${{ inputs.node_version }}
+        if: inputs.docker_compose_file == ''
       - name: Setup Docker Buildx
         id: buildx
         uses: docker/setup-buildx-action@v3
         with:
-          buildkitd-flags: '--debug'
+          buildkitd-flags: "--debug"
+        if: inputs.docker_compose_file != ''
```

Comment:
I think we should always setup buildx do the builder is available to test-containers?

Reply:
ok fair